src/coffee/helpers/user-agent.coffee

```diff
@@ -1,9 +1,9 @@
 { arch, platform } = require 'os'
-getPkg = require 'load-module-pkg'
+{ SphereClient } = require 'sphere-node-sdk'
 
 module.exports = (pkgName, pkgVersion) =>
   throw new Error 'Package name and version is required' unless pkgName and pkgVersion
-  sdkPkgVersion = getPkg.sync('sphere-node-sdk').version
+  sdkPkgVersion = SphereClient.getVersion
```

Comment:
Im just wondering if we dont need to invoke this method? 
`SphereClient.getVersion()`

Reply:
right this can't work, even in CoffeeScript.. will fix it

Reply:
clearly the unit tests don't work either